SYCL/Basic/zero_size_range.cpp

```diff
@@ -0,0 +1,17 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
This line only builds the test. So, currently test it is not executed.
Could you please add lines to execute the test?